db_test.go

```diff
@@ -27,14 +27,14 @@ type testUser struct {
 }
 
 // getOptions returns a list of options to be used when creating a new websocket connection
-func getOptions() (options []surrealdb.SurrealDBOption) {
-	// Set option to timeout after 20 seconds.
-	optionfunc := func(ws *websocket.WebSocket) error {
-		ws.Timeout = time.Duration(20) * time.Second
-		ws.Conn.EnableWriteCompression(true)
-		return nil
+func getOptions() (options []surrealdb.Option) {
+	optWriteCompression := surrealdb.Option{
+		WsOption: func(ws *websocket.WebSocket) error {
+			ws.Conn.EnableWriteCompression(true)
```

Comment:
If this is something we want to support setting I can also add an option for this.

Reply:
From docs
> EnableWriteCompression enables and disables write compression of subsequent text and binary messages. This function is a noop if compression was not negotiated with the peer.

I say yes and let's enable it by default. Thank you for this @plally !